pkg/processtree/processtree.go

```diff
@@ -83,22 +83,18 @@ type ProcessTreeCollector interface {
 	Events() <-chan ProcessTreeEvent
 }
 
-type containerClient interface {
-	LoadContainerTasks(ctx context.Context) ([]containers.ContainerProcess, error)
-}
-
 type ProcessTreeCollectorImpl struct {
 	log              *logging.Logger
 	proc             *proc.Proc
-	containersClient containerClient
+	initialProcesses []containers.ContainerProcess
 	eventSink        chan ProcessTreeEvent
 }
 
-func New(log *logging.Logger, p *proc.Proc, containersClient containerClient) (*ProcessTreeCollectorImpl, error) {
+func New(log *logging.Logger, p *proc.Proc, initialProcesses []containers.ContainerProcess) (*ProcessTreeCollectorImpl, error) {
```

Comment:
I am not sure about this change. It feels a bit weird to pass the initial processes to the process tree to be honest. I would rather stick with the client. 

Also a side effect of this is, that the GC cannot cleanup the slice 😅 Since the initial processes are only ever needed during `Init`, I guess instead we can pass it in there, or just stick to the container client.

Reply:
Good point, in first version I needed this globally, but now it's a waste, will fix.